examples/trials/efficientnet/README.md

```diff
@@ -0,0 +1,33 @@
+# EfficientNet
+
+[arXiv](https://arxiv.org/abs/1905.11946)
+
+We provide here:
+
+* Search space and tuners for finding the best tuple (alpha, beta, gamma) for EfficientNet-B1 with
+  grid search, as discussed in Section 3.3 in [paper](https://arxiv.org/abs/1905.11946).
+* Search space for EfficientNet transfer learning on datasets other than ImageNet;
+  settings are borrowed from Appendix A.5 from 
+  "[Do Better ImageNet Models Transfer Better?](https://arxiv.org/abs/1805.08974)".
+
+## Instructions
+
+1. Clone our modified version of [EfficientNet-PyTorch](https://github.com/ultmaster/EfficientNet-PyTorch) in this directory.
```

Comment:
what's the difference between the modified version and the original version?

Reply:
documented